pkg/controller/feature-gates.go

```diff
@@ -0,0 +1,59 @@
+package controller
+
+import (
+	"context"
+	"k8s.io/apimachinery/pkg/types"
+	cdiv1 "kubevirt.io/containerized-data-importer/pkg/apis/core/v1beta1"
+	"kubevirt.io/containerized-data-importer/pkg/common"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sync"
+)
+
+const (
+	// SkipWaitForFirstConsumerVolumes - if enabled will not schedule worker pods on a storage with WaitForFirstConsumer binding mode
+	SkipWaitForFirstConsumerVolumes = "SkipWaitForFirstConsumerVolumes"
```

Comment:
Is it just me, or could there be a better name?  Like `HonorWaitForFirstConsumer`?

Reply:
Or Maybe even just `WaitForFirstConsumer`, name should be good, because once set it will stay for some time.

Reply:
now as I read this, `HonorWaitForFirstConsumerEnabled` is much better than Skip - done